src/fiatconnect/saga.ts

```diff
@@ -50,6 +68,67 @@ export function* watchFetchFiatConnectQuotes() {
   yield takeLeading(fetchFiatConnectQuotes.type, handleFetchFiatConnectQuotes)
 }
 
+export function* handleStartFiatConnectTransfer({
+  payload: params,
+}: ReturnType<typeof startFiatConnectTransfer>) {
+  const { flow, quoteId, fiatConnectQuote, fiatAccountId } = params
+
+  if (flow === CICOFlow.CashOut) {
+    try {
+      Logger.info(TAG, 'Starting transfer out..')
+      const transferResult: TransferResponse = yield call(
+        doTransferOut,
+        fiatConnectQuote,
+        fiatAccountId
+      )
+
+      Logger.info(
+        TAG,
+        'Transfer out $succeeded. Starting transaction..',
+        JSON.stringify(transferResult)
+      )
+
+      const tokenList: TokenBalance[] = yield select(tokensListSelector)
+      const tokenInfo = tokenList.find((token) => token.symbol === fiatConnectQuote.getCryptoType())
+
+      if (!tokenInfo) {
```

Comment:
There's no situation where this ever ought to occur, right? Maybe add a comment clarifying this?